app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -980,6 +980,7 @@ BrowserWorld::InitializeJava(JNIEnv* aEnv, jobject& aActivity, jobject& aAssetMa
     });
 
     UpdateEnvironment();
+    VRBrowser::CheckTogglePassthrough();
```

Comment:
This is likely a suggestion for a follow up patch, but I wonder whether we should call this function before the UpdateEnviroment, and introduce additional logic to avoid creating the skybox if the device is started with the passthrough switch on.